include/deal.II/base/numbers.h

```diff
@@ -37,8 +37,55 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+namespace internal
+{
+  template <typename Number>
+  struct VectorizedArrayWidthSpecifier
+  {
+    static const unsigned int max_width = 1;
```

Comment:
This and the corresponding variables in the specializations could be made `constexpr`.